src/pages/messages/components/NviCandidateActions.tsx

```diff
@@ -169,14 +169,14 @@ export const NviCandidateActions = ({ nviCandidate, nviCandidateQueryKey }: NviC
               gap: '0.25rem',
             }}>
             {sortedNotes.map((note) => {
-              let deleteFunction: (() => void) | undefined = undefined;
+              let deleteFunction: (() => Promise<any>) | undefined = undefined;
```

Comment:
skal sikkert ikke brukes igjen så gjør jo ikke noe, men siden det var riktig type før:
```suggestion
              let deleteFunction: (() => Promise<void>) | undefined = undefined;
```